docs/api/sql/aggregate-functions.md

```diff
@@ -99,6 +99,12 @@ Create an aggregate function that calculates the sum of the input _expression_.
 
 Create an aggregate function that calculates the product of the input _expression_.
 
+## geomean
+
+`geomean(expression)`
```

Comment:
should this be `geomean` or `geoMean` to be consistent with others?

Reply:
I vote for `geomean` (keep it as-is)